packages/anomaly-detection/src/types/IAnomalyDetection.ts

```diff
@@ -0,0 +1,16 @@
+import {IAgentContext, IPluginMethodMap} from '@veramo/core'
+import {AnomalyDetectionStoreArgs, IAnomalyDetectionStore} from "@sphereon/ssi-sdk.anomaly-detection-store";
+import {Optional} from "nx/src/project-graph/plugins";
```

Comment:
/src import

Reply:
It looks like a mixed it up with Java :see_no_evil:

Reply:
Replaced it with a Partial